WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteItem.kt

```diff
@@ -43,6 +48,29 @@ sealed class MySiteItem(val type: Type) {
 
     data class DomainRegistrationBlock(val onClick: ListItemInteraction) : MySiteItem(DOMAIN_REGISTRATION_BLOCK)
 
+    data class QuickStartCard(
+        val id: String,
+        val title: String,
+        val tasks: List<DummyTask>,
+        val progressColor: ProgressColor,
+        val onMoreClick: ListItemInteraction? = null
+    ) : MySiteItem(QUICK_START_CARD) {
+        val doneTasks = tasks.filter { it.done }
+        val progress = if (tasks.isNotEmpty()) ((doneTasks.size / tasks.size.toFloat()) * 100).roundToInt() else 0
+
+        enum class ProgressColor(@ColorRes val indicatorColor: Int, @ColorRes val trackColor: Int) {
+            GREEN(R.color.green_20, R.color.green_0),
+            ORANGE(R.color.orange_40, R.color.orange_0)
+        }
+
+        data class DummyTask(
```

Comment:
I think we can replace the "Dummy" names for the object and view holder with something we'll use later, right? They can easily be called something like "QuickStartTask" or something similar. Even if they are just dummy objects now, WDYT?

Reply:
That's right. My intention was always to replace those dummy objects with real implementations later on, but I decided to do that while handling #13584 since it seemed to be more related to that part of the work and also because having dummy objects here would give me more space to focus on the parent card UI without committing to a specific approach for the children cards UI. Ultimately, I'll end up reutilizing more things than I expected, so I could get rid of the dummy objects here, but I still prefer to keep them for now, just because I'm about to submit another PR with these changes anyway.